file_catalog/server.py

```diff
@@ -83,20 +82,23 @@ def set_last_modification_date(d):
 class Server(object):
     """A file_catalog server instance"""
 
-    def __init__(self, config, port=8888, db_host='localhost', db_port=27017, debug=False):
+    def __init__(self, config, port=8888, debug=False,
+            db_host='localhost', db_port=27017, db_user=None, db_pass=None):
         static_path = get_pkgdata_filename('file_catalog', 'data/www')
         if static_path is None:
             raise Exception('bad static path')
         template_path = get_pkgdata_filename('file_catalog', 'data/www_templates')
         if template_path is None:
             raise Exception('bad template path')
 
-        # print configuration
         logger.info('db host: %s', db_host)
         logger.info('db port: %s', db_port)
+        logger.info('db user: %s', db_user)
         logger.info('server port: %r', port)
         logger.info('debug: %r', debug)
-        logger.info('config: %r', config)
+        redacted_config = dict(config)
```

Comment:
Not really necessary in this specific case, but consider:
`redacted_config = copy.deepcopy(config)`